flixel/input/gamepad/id/FlxGamepadPointerValueList.hx

```diff
@@ -0,0 +1,29 @@
+package flixel.input.gamepad.id;
+import flixel.input.gamepad.FlxGamepad;
+
+/**
+ * A helper class for gamepad input.
+ * Provides optimized gamepad button checking using direct array access.
+ */
+@:keep
+class FlxGamepadPointerValueList
+{
+	private var gamepad:FlxGamepad;
+	
+	@:allow(flixel.input.gamepad.FlxGamepad)
+	public var available(default, null):Bool = true;
```

Comment:
Might make sense to change `available` to `isSupported`? That's what `FlxAccelerometer` uses -> consistent API.